rllib/algorithms/algorithm_config.py

```diff
@@ -223,7 +223,7 @@ def __init__(self, algo_class=None):
         self.sample_collector = SimpleListCollector
         self.create_env_on_local_worker = False
         self.sample_async = False
-        self.enable_connectors = False
+        self.enable_connectors = True
```

Comment:
😄

Reply:
I've purposely done this for now to check if all text_external_env.py tests pass. 

They were hanging in my local machine